src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
It is true that `BSONObj` is not like `string`, as BSON already is some kind of prefix that clearly ties it to mongo.
So, if you really want to, I'll do it 'your way' in this file.
My personal vote is still for leaving it the way it is.
When namepaces become really long, like over 15 chars, then I will probably have another opinion :-)
Right here it is not so important (due to the prefix in BSONObj - had it been mongobson::Obj instead, it would be very different).